proto/options.proto

```diff
@@ -119,6 +119,7 @@ message CostingOptions {
   optional bool ignore_access = 62;
   optional bool ignore_closures = 63;
   optional bool shortest = 64;
+  optional uint32 max_speed = 65;
```

Comment:
can we name this top_speed like the parameter is called in the request? also maxspeed reminds me of the osm tag for speed limit so top speed is a good to differentiate

Reply:
Oops, didn't see that was still in there.. Yeah, I only realized that `top_speed` was already there after a few minutes.. Will revert.

Reply:
yep no worries!